app/0.10.x/admin-api.md

```diff
@@ -1914,3 +1914,6 @@ HTTP 204 No Content
 [clustering]: /{{page.kong_version}}/clustering
 [cli]: /{{page.kong_version}}/cli
 [secure-admin-api]: /{{page.kong_version}}/secure-admin-api
+
+
+Notice that when matching a request against an api's hosts field, the match is case sensitive.
```

Comment:
While this statement may be true, it is in an incorrect place in the docs - please move it to the correct location.

Additionally, please add this clarifying note to the other version's of Kong's documentation that need it (this change is only on Kong 0.10 docs - it likely applies to other versions of Kong)

Reply:
Line 8 would likely be a better place to add the note.